docs/data/material/components/slider/CustomMarks.js

```diff
@@ -16,35 +16,38 @@ const marks = [
   },
 ];
 
-const Label = styled('div')(() => ({
+const labelStyle = {
   fontSize: '0.875rem',
-  lineHeight: '1.5',
-  fontWeight: 500,
+  lineHeight: '1.43',
+  fontWeight: 400,
   cursor: 'pointer',
-}));
+};
```

Comment:
I don't think we need these custom typography styles — we can probably pass all of that with native props on the Typography component:

```
<Typography variant="body2" />```

And that's it :)

Reply:
That was merely to make the custom label look similar to the default marks from default `Slider`, but since this is under customization, I believe we can make it a bit different